tools/CleanOutput.py

```diff
@@ -0,0 +1,77 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+import os
+import re
+import logging
+
+
+def clean_output(input_file, output_dir):
+    """
+    Deletes all output files generated by a run with the `input_file` in
+    the `output_dir`.
+
+    The `input_file` must list its expected output files in a comment:
```

Comment:
Could you add a note saying that there must be exactly two spaces of indentation after `ExpectedOutput`?